projects.md

```diff
@@ -0,0 +1,67 @@
+# Projects
+
+## Sales Prospect & Internal Proposals
+
+While the Business Planning WG is mainly in charge of sales lead generation,
+all members may pursue leads as they come up.
+These may be a client or a grant opportunity.
+Members are strongly encouraged to record leads into the co-operative's [leads tracking system](https://link.hypha.coop/leads) whether or not we want to pursue this lead.
+
+If a member decides to pursue a **lead**,
+they can reach out for preliminary discussions to see whether there is a potential to work together.
+If the lead expresses intention to proceed with a concrete project in mind,
```

Comment:
A prospect is just willing to listen, and so you get a discovery call with them to qualify them. "intention to proceed with a concrete project in mind" seems a lot stronger than i understood it to be? You words describe a "qualified prospect" / "opportunity", I believe.

See: https://blog.hubspot.com/sales/ultimate-guide-to-sales-qualification

What about one of these:

> - If the lead _seems interested in working together_, this becomes a **prospect**.
>
> - If the lead expresses intention to proceed with a concrete project in mind, the lead becomes more **highly qualified**, becoming an **opportunity**.

Reply:
But you can't draft an internal proposal if someone "seems interested in working together", only when there is a specific project in mind?

Reply:
Yes, then the second suggestion fits. According to your description, you wouldn't be working on a prospect if you're writing an internal proposal in your flow. Will submit PR suggestion directly

Reply:
Agree this phrasing is awk and something in it doesn't map to my understanding of how we're using these terms

Reply:
This was originally not written with the "established?" sales definitions of `lead`, `prospect`, `qualified prospect`, `opportunity` and I feel like we are trying to modify the text to conform to some definition I don't have a good grasp on, and to be honest I am not sure if necessary at our stage. I will leave this language to @patcon and @dcwalk to modify as suitable.

Reply:
I believe this was resolved by using "opportunity" instead of "prospect"